portal-api/src/v1/api.js

```diff
@@ -268,23 +268,35 @@ const saveUtilisationReport = async (reportData, month, year, user, filePath) =>
   }
 };
 
-const getUtilisationReports = async (bankId) => {
+const getUtilisationReports = async (bankId, month, year) => {
   try {
     if (!isValidBankId(bankId)) {
       console.error('Get utilisation reports failed with the following bank ID %s', bankId);
       return false;
     }
 
+    if (month && !isValidMonth(parseInt(month, 10))) {
+      console.error('Get utilisation reports failed with the following month %s', month);
+      throw new Error('Invalid month provided: %s', month);
+    }
+
+    if (year && !isValidYear(parseInt(year, 10))) {
+      console.error('Get utilisation reports failed with the following year %s', year);
+      throw new Error('Invalid year provided: %s', year);
+    }
+
+    const queryString = month ? `?month=${month}${year ? `&year=${year}` : ''}` : '';
+
     const response = await axios({
       method: 'get',
-      url: `${DTFS_CENTRAL_API_URL}/v1/portal/previous-reports/${bankId}`,
+      url: `${DTFS_CENTRAL_API_URL}/v1/bank/${bankId}/utilisation-reports${queryString}`,
       headers: headers.central,
     });
```

Comment:
I think you could use the `param` config attribute in the axios request to set the query params instead of hand rolling it. So this could instead be:
```
const response = await axios.get(`${DTFS_CENTRAL_API_URL}/v1/bank/${bankId}/utilisation-reports`, {
  params: { month, year },
  headers: headers.central,
});
```

Reply:
Ah didn't know this existed, so much nicer!